controllers/StockApiController.php

```diff
@@ -626,7 +626,7 @@ public function ProductPrintLabel(\Psr\Http\Message\ServerRequestInterface $requ
 			(new WebhookRunner())->run(GROCY_LABEL_PRINTER_WEBHOOK, $webhookData, GROCY_LABEL_PRINTER_HOOK_JSON);
 		}
 
-		return $this->ApiResponse($response, $webhookData);
+		return $this->EmptyApiResponse($response);
```

Comment:
Are you sure this change does not break local (in-browser) running of the webhook? Because the webhook data is needed by client-side javascript when printing from the click trigger (context menus of products / stock entries) in public/viewjs/stockoverview.js (or in the case below from public/viewjs/stockentries.js).

Still works when running server-side though, and won't be needed if I continue down the modern js path because product / stockentry data would be available anyways.

Reply:
Sorry, just thought at first why there needs to be a response at that API endpoint but haven't tested the client side webhook running afterwards...

Reverted in https://github.com/grocy/grocy/commit/fe59fac1c374aec0357e85a20a6925868a2df2bc.

Reply:
No worries! We catched it before release and that's all what matters. Thank you for the quick fix :)